src/open_inwoner/templates/pages/search.html

```diff
@@ -20,10 +20,26 @@ <h1 class="h1">Zoeken</h1>
         {#  facets and results section #}
         {% if results %}
             <div class="grid">
-                <aside class="grid__sidebar" aria-label="{% trans "Zoekfilters" %}">
-                    {% filter field=form.categories %}
-                    {% filter field=form.tags %}
-                    {% filter field=form.organizations %}
+                <aside class="grid__sidebar grid__filters" aria-label="{% trans "Zoekfilters" %}">
+                    <h2 class="h2">{% trans "Zoekfilters" %}</h2>
+                    {% if search_filters %}
+                        {% if search_filter_categories %}
+                            {% if form.categories|length > 0 %}
```

Comment:
This `|length > 0` looks unusual. 

If you evaluate `form.somefield` in a template then Django will in fact render the html output, eg: ``{{ form.somefield }}``. 

So what this line does is render the default widget, then check if the html string has any length, and if so render it again using our custom widgets. That is not efficient and probably not what we really want to do here.

Reply:
Yes, with this weirdness I was trying to not display the ugly empty list after search (checkboxes are currently completely removed if there are no results for filters that need to exist) - but with Alex decided to move this behaviour to a new issue
(https://taiga.maykinmedia.nl/project/open-inwoner/task/1749).